starboard/tools/testing/test_runner.py

```diff
@@ -180,7 +180,7 @@ def Start(self):
           assert hasattr(self.launcher, "CheckPackageIsDeployed")
           if abstract_launcher.ARG_NOINSTALL not in self.launcher.launcher_args:
             self.launcher.Deploy()
-          if not self.launcher.CheckPackageIsDeployed():
+          if not self.launcher.CheckPackageIsDeployed("GoogleInc.YouTube"):
```

Comment:
Can you make this the default argument in the xb1 launcher instead? While this is probably only used for xb1 today I'm not sure it's gonna stay that way.

Reply:
Done. I also caught another issue where I previously used the wrong method name, and fixed that.